src/main/java/CommandLine/Parser.java

```diff
@@ -0,0 +1,106 @@
+package CommandLine;
+
+import Task.TaskList;
+import Task.Task;
+
+import static java.lang.Integer.parseInt;
+
+public class Parser {
+    public Parser() {
+
+    };
+
+    /**
+     * Parses the input supplied by user, adds it to the task list if possible
+     * This method helps to parse inputs, calling the appropriate commands where possible
```

Comment:
nit: There should be a full stop at the end of the method description, and a line to separate the description and the method parameters. I noticed this in a few other places as well.